app/forms/location_form.rb

```diff
@@ -65,6 +65,7 @@ def destroy(params)
   end
 
   def check_user
+    errors.add(:base, "Technician's cannot create/edit protected locations") if @current_user.technician? && @params[:location][:protect] == "1" # protect = true
```

Comment:
In theory when editing a location, technicians could make a location go from protected -> unprotected and this check would not fail, however it is not possible to check this unless we check the initial location values then determine what has changed

Reply:
I don't quite follow your comment, maybe we could talk this one through. 
Could this be added to the user_permission relevant model
Or maybe even pulling this into its own validation/ method like [here](https://stackoverflow.com/questions/42495133/rails-validation-if-one-condition-is-true/42495298) so the specific `if` isnt being called in the generic `check_user` methig